src/hooks/useCioAutocomplete.ts

```diff
@@ -189,7 +189,7 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
           openMenu();
         }
         try {
-          if (advancedParameters?.fetchZeroStateOnFocus && items?.length === 0) {
+          if (advancedParameters?.fetchZeroStateOnFocus) {
```

Comment:
If you revert this change and test the storybook example you added everything will work the same. The recommendation pod get fetched when the filters change already I'm not even sure there's a problem to fix here 🤔 @ZSnake